pgportfolio/learn/network.py

```diff
@@ -97,8 +97,9 @@ def _build_network(self, layers):
                 network = network[:, :, 0, 0]
                 #btc_bias = tf.zeros((self.input_num, 1))
                 btc_bias = tf.get_variable("btc_bias", [1, 1], dtype=tf.float32,
-                                       initializer=tf.zeros_initializer)
+                                       initializer=tf.ones_initializer)
```

Comment:
Why `ones_initializer` is preferred there? Won't the weight be more than 1?

Reply:
I did this as a test and forgot to change it back. I'll update the PR to convert this back to `zeros_initializer`.